tmp-announcement.md

```diff
@@ -3,34 +3,31 @@ This will be created as an issue in all collection repositories mentioned in htt
+* The list of requirements for inclusion in Ansible: https://github.com/ansible-collections/overview/blob/master/collection_requirements.rst
+* The roadmap with all important dates for Ansible 2.10: https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst
 
-The next sections contain the more important points for now.
+## DEADLINE: 2020-08-18
 
-## Important date: 2020-08-18
+The latest version of the collection available on August 18 will be included in Ansible 2.10.0, except possibly newer versions which differ only in the patch level. (For details, see [the roadmap](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst)). Please release version 1.0.0 of your collection by this date! **If 1.0.0 does not exist, the same 0.x.y version will be used in all of Ansible 2.10 without updates,** and your 1.x.y release will not be included until Ansible 2.11 (unless you request an exception at a [community working group meeting](https://github.com/ansible/community/issues/539) and go through a demanding manual process to vouch for backwards compatibility . . . you want to avoid this!).
 
-The latest version of the collection by this date will be used in Ansible 2.10.0, except possibly newer versions which differ only in the patch level. (For details, see [the roadmap](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst)). In particular, by this date, version 1.0.0 of your collection should be released. **If not, the same 0.x.y version will be used in all of Ansible 2.10 without updates,** and a 1.x.y will only be included in Ansible 2.11 (except if you can vouch for backwards compatibility with the included version, and then it's a manual process we really want to avoid).
+## Follow semantic versioning rules
 
-## Your collection versioning must stick to the semantic versioning rules
-
-This means:
+Your collection versioning must follow all [semver rules](https://semver.org/). This means:
 
 * Patch level releases can only contain bugfixes;
 * Minor releases can contain new features, new modules and plugins, and bugfixes, but must not break backwards compatibility;
 * Major releases can break backwards compatibility.
 
-The detailed rules can be found [here](https://semver.org/).
-
 ## Changelogs and Porting Guide
 
-For Ansible 2.10, the changelog and porting guide are automatically generated from the changelog and porting guide of ansible-base, and from the changelogs of the included collections.
+Your collection should provide fragments for the Ansible 2.10 changelog and porting guide. The changelog and porting guide are automatically generated from ansible-base, and from the changelogs of the included collections. All fragments marked `breaking_changes`, `major_changes`, `removed_features` and `deprecated_features` will appear in both the changelog and the porting guide. You have two options for providing changelog fragments to include:
```

Comment:
```suggestion
Your collection should provide data for the Ansible 2.10 changelog and porting guide. The changelog and porting guide are automatically generated from ansible-base, and from the changelogs of the included collections. All changes from the `breaking_changes`, `major_changes`, `removed_features` and `deprecated_features` sections will appear in both the changelog and the porting guide. You have two options for providing changelog fragments to include:
```

Reply:
We don't care about changelog fragments, these are part of how the changelog is created, not of how we can process it :)